newsletter_automation/newsletter/routes.py

```diff
@@ -512,3 +515,4 @@ def prometheus_metrics():
     cpu_usage.set(cpu_percent)
```

Comment:
Are you overwriting the prometheus metric? I did not follow the change to this method. I was assuming you would not touch this method except to say do not track this method.

Reply:
I am not touching this method. I'm just telling flask exporter to ignore tracking this endpoint. No other changes